editor/selectors.js

```diff
@@ -37,6 +38,20 @@ export function getEditedPostTitle( state ) {
 		: state.editor.edits.title;
 }
 
+export function getEditedPostPreviewLink( state ) {
+	const link = state.currentPost.link;
+	if ( ! link ) {
+		return null;
+	}
+
+	const queryStringPosition = link.indexOf( '?' );
```

Comment:
>If we were to add this kind of utilities that could be useful outside the `editor` project, should we create a new root folder `utils` or something like that?

Yeah, I agree it's generally useful. `utils` feels a little generic so far as a reconciliation path is concerned. There is a [`wp-utils.js`](https://github.com/WordPress/WordPress/blob/master/wp-includes/js/wp-util.js) file in core already which exposes a couple relatively specific properties on the `wp` global. In [Trac#38342](https://core.trac.wordpress.org/ticket/38342)'s latest patch we'd proposed adding `wp.formatting`, and elsewhere `wp.date` in [Trac#39222](https://core.trac.wordpress.org/ticket/39222). Something like `wp.url`, `wp.formatting.addQueryArgs`, or `wp.formatting.url.addQueryArgs`? Just throwing out ideas at this point 😄